### PR TITLE
Classic V2

### PR DESCRIPTION
**Pull request by luishdz1010**
- Uses Sage as base to allow including WP, plugins and libraries using composer
- Removes wp db plugin (there is no free, updated plugin version anymore)
- Do not commit lock files so that new install pull newer dependencies
- You can see the folder structure at https://github.com/wearesideways/sw-classic-skeleton/tree/lh/v2

### CODE REVIEW

File: buildspec.yml [hunk 1]
```diff
@@ -1,33 +1,40 @@
...
+  install:
+    runtime-versions:
+      php: 7.4
+      nodejs: 14
+
   pre_build:
     commands:
-      - cp wp-config-server-tpl.php wp-config.php
-      - cd wp-content/themes/<REPLACE_ME>/
+      - composer install --no-progress --no-dev --no-suggest
```
Comment (luishdz1010) [inline buildspec.yml:11]: Composer is run during CI

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,7 @@
+DB_HOST='localhost'
```
Comment (luishdz1010) [inline .env.example:1]: Simple env file, we use configs overrides per environment (as in HOSS)

File: .ebextensions/eb.config [hunk 3]
```diff
@@ -0,0 +1,8 @@
+files:
+  "/etc/php.d/99-wordpress.ini" :
```
Comment (luishdz1010) [inline .ebextensions/eb.config:2]: Adds ability to change PHP settings per instance

File: .platform/nginx/conf.d/elasticbeanstalk/00-wordpress.conf [hunk 4]
```diff
@@ -0,0 +1,51 @@
+add_header X-Frame-Options "SAMEORIGIN";
```
Comment (luishdz1010) [inline .platform/nginx/conf.d/elasticbeanstalk/00-wordpress.conf:1]: We now host on nginx (this is true also for current sites on PHP 7.4, pending production push)

File: NEW_PROJECT_INSTALLATION.md [hunk 5]
```diff
@@ -1,40 +1,36 @@
...
-### Do Replaces
-Replace all the `<REPLACE_ME>` strings by the proper ones under the following files:
-- `buildspec.yml` file
-- `wp-content/themes/theme/build-settings.js` file
-- `wp-content/themes/theme/style.css` file
-- `wp-content/themes/theme/core/acf-configuration.php` file
-- `wp-content/themes/theme/core/ses-mailer.php` file
+Replace all instances of SW_PROJECT with the project's name:
 
+1. Replace all files and folders: `find . -depth -name '*SW_PROJECT*' -execdir bash -c 'for f; do mv -i "$f" "${f//SW_PROJECT/PUT_THE_PROJECT_NAME_HERE}"; done' bash {} +`
```
Comment (luishdz1010) [inline NEW_PROJECT_INSTALLATION.md:7]: Can someone check these 2 cmds on mac? @gera919 @jaimeglez

File: composer.json [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://wpackagist.org",
+      "only": ["wpackagist-plugin/*", "wpackagist-theme/*"]
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "advanced-custom-fields/advanced-custom-fields-pro",
```
Comment (luishdz1010) [inline composer.json:13]: ACF Pro gets auto updated as well!

File: phpcs.xml [hunk 7]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
```
Comment (luishdz1010) [inline phpcs.xml:1]: we should setup IDE support for this, and also run the PHP linter on build

**Review by jaimeglez — APPROVED** [review]

**Review by gera919 — APPROVED** [review]

**Pull request merged**